WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3308,6 +3295,69 @@ public void onTrackableEvent(TrackableEvent event, Map<String, String> propertie
         }
     }
 
+    @Override public void onRetryUploadForMediaCollection(ArrayList<Object> mediaFiles) {
```

Comment:
Hi @mzorz could this be moved to another class for testability and line reduction reasons? LMK 👍

Reply:
👍 addressed this request in this PR https://github.com/wordpress-mobile/WordPress-Android/pull/13198 targeting this one here (so that one should be reviewed and merged first, then you could continue reviewing this one properly here)

Reply:
Nice. This is such a great improvement 👍